### PR TITLE
Try to fix bedtools_map 

### PR DESCRIPTION
**Pull request by bgruening**
https://github.com/galaxyproject/tools-iuc/issues/1447

### CODE REVIEW

File: tools/bedtools/mapBed.xml [hunk 1]
```diff
@@ -8,16 +8,16 @@
...
         $strand
         @C_AND_O_ARGUMENT@
         -f $overlap
         $reciprocal
         $split
         $header
-        #if $genome.genome_choose == "-g" :
-            -g $genome.genome
+        #if $genome_select.genome_choose == "-g" :
+            -g $genome_select.genome
```
Comment (nsoranzo) [inline tools/bedtools/mapBed.xml]: Add single quotes.

File: tools/bedtools/mapBed.xml [hunk 2]
```diff
@@ -35,9 +35,13 @@
         </expand>
         <expand macro="split" />
         <expand macro="print_header" />
-        <conditional name="genome">
-            <param name="genome_choose" type="boolean" checked="false" truevalue="-g" falsevalue=""
-                label="Treat split/spliced BAM or BED12 entries as distinct BED intervals when computing coverage." help="(-g)" />
+        <conditional name="genome_select">
+            <param name="genome_choose" type="select" 
+                   label="Treat split/spliced BAM or BED12 entries as distinct BED intervals when computing coverage"
+                   help="(-g)" >
```
Comment (nsoranzo) [inline tools/bedtools/mapBed.xml]: `argument` instead of `help`.

**Comment by nsoranzo** [comment]
This will conflict with https://github.com/galaxyproject/tools-iuc/pull/1446

**Comment by FredericBGA** [comment]
thank you.
could you please help me to merge this (using git) with my own modifications here #1446

**Comment by FredericBGA** [comment]
fixed I hope in #1446

**Comment by nsoranzo** [comment]
Superseded by #1446.

**Pull request closed without merging**